src/AutoSysimages.jl

```diff
@@ -156,7 +160,8 @@ function build_sysimage(background::Bool = false)
         if isinteractive()
             @info """AutoSysimages: No project settings (SysimagePreferences.toml) found.
 Do you want to select packages to be included now?"""
-            if request(RadioMenu(["Yes (select packages)", "No (include all packages)"])) == 1
+            if request(RadioMenu(["Yes (select packages)", "No (include all packages)"])) ==
+               1
```

Comment:
Thanks for formatting. However, here, i wouldn't split the line not to have a single symbol on the next line.

Reply:
Ok, i will no more format on this repo then ;)